package.json

```diff
@@ -61,6 +60,28 @@
     "build:dist": "npm run clean:dist && BABEL_ENV=build babel src --out-dir dist --copy-files --ignore spec.js,example.js,.md",
     "build": "npm run test && npm run build:examples && npm run build:docs && npm run build:js && npm run build:dist"
   },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "jsx"
+    ],
+    "moduleDirectories": [
+      "node_modules"
+    ],
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/jest/__mocks__/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/jest/__mocks__/styleMock.js"
+    },
+    "transformIgnorePatterns": [
+      "node_modules/(?!ol)"
+    ],
+    "setupFiles": [
+      "<rootDir>/jest/__mocks__/shim.js",
+      "<rootDir>/jest/setup.js"
+    ],
+    "collectCoverage": false,
```

Comment:
We can probably just override this flag when we are running in a CI envoronment, an then the rest of the coveralls interaction is mostly the same.